specification/orbital/resource-manager/Microsoft.Orbital/aodg/preview/2024-03-01-preview/aodg.json

```diff
@@ -0,0 +1,1672 @@
+          "readOnly": true
+        }
+      },
+      "description": "Response for an API service call that lists the resource IDs of resources associated with another resource."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Creating",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Updating",
+        "Deleting"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      },
+      "description": "The current state of the resource's creation, deletion, or modification.",
+      "readOnly": true
+    },
+    "GroundStationsProperties": {
+      "type": "object",
+      "required": [
+        "capabilities",
+        "globalCommunicationsSite"
+      ],
+      "properties": {
+        "city": {
+          "type": "string",
+          "description": "City of ground station."
+        },
+        "capabilities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Capability"
+          },
+          "description": "Ground station capabilities."
+        },
+        "providerName": {
+          "type": "string",
+          "description": "Ground station provider name."
+        },
+        "longitudeDegrees": {
+          "type": "number",
+          "description": "Longitude of the ground station in decimal degrees."
+        },
+        "latitudeDegrees": {
+          "type": "number",
+          "description": "Latitude of the ground station in decimal degrees."
+        },
+        "altitudeMeters": {
+          "type": "number",
+          "description": "Altitude of the ground station."
+        },
+        "releaseMode": {
+          "type": "string",
+          "enum": [
+            "Preview",
+            "GA"
+          ],
+          "x-ms-enum": {
+            "name": "releaseMode",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "Release Status of a ground station."
+        },
+        "globalCommunicationsSite": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/ResourceReference"
+            }
+          ],
+          "description": "A reference to global communications site."
+        }
+      },
+      "description": "List of ground station resource properties."
+    },
+    "ProvisionedBandwidth": {
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "mbps": {
+          "type": "integer",
+          "format": "int32",
+          "minimum": 0,
+          "readOnly": true,
+          "description": "The approximate number of megabits per second that are provisioned for the partner on this global communications site."
+        }
+      },
+      "description": "The approximate bandwidth provisioned for a partner on a global communications site."
+    },
+    "ResourceReference": {
+      "type": "object",
+      "required": [
+        "id"
```

Comment:
>id [](http://example.com/codeflow?start=9&length=2)

Properties in the property bag are not allowed to duplicate top-level ARM property names (e.g. id, type, name). If this reference could refer to different types, then you can call it "resourceId".

Reply:
We share the resource with orbital common infra, this resource is a part of that infra.
FYI https://github.com/Azure/azure-rest-api-specs/blob/b72d96e65d5965befbde5559508fb4eb7cb942db/specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json#L1252C5-L1264C7

If we change this we have to change our stable version too.